Tests/AI.cpp

```diff
@@ -155,4 +156,87 @@ TEST(ai_test, computeBestWrest){
         gameController::WrestQuaffle wrest(env, player, env->quaffle->position);
         EXPECT_NE(wrest.check(), gameController::ActionCheckResult::Impossible);
     }
+}
+
+//------------------------------------------Interference Useful Tests--------------------------------------------------
+
+TEST(ai_test, getNextFanTurn0){
+    auto env = setup::createEnv();
+    communication::messages::broadcast::Next next = communication::messages::broadcast::Next{communication::messages::types::EntityId::LEFT_GOBLIN, communication::messages::types::TurnType::FAN, 30};
```

Comment:
Einfach direkt den Ctor aufrufen
```suggestion
    communication::messages::broadcast::Next next{communication::messages::types::EntityId::LEFT_GOBLIN, communication::messages::types::TurnType::FAN, 30};
```